src/sickle.ts

```diff
@@ -146,12 +168,14 @@ class Annotator {
    * Returns empty string if there is no existing annotation.
    */
   private existingClosureAnnotation(p: ts.PropertyDeclaration | ts.ParameterDeclaration) {
-    let comments = ts.getLeadingCommentRanges(this.file.text, p.pos);
+    let text = p.getFullText();
+    let comments = ts.getLeadingCommentRanges(text, 0);
```

Comment:
odd, but indeed it is harmless. That whole getLeadingCommentRanges api is very odd. I don't know why they dont' have p.getLeadingCommentRanges without option for passing text (and thus getting it wrong by not using fulltext).